src/components/timeline/TimelineViewControls.svelte

```diff
@@ -201,6 +260,21 @@
   <LinkIcon />
 </button>
 
+<ToggleableIconButton
+  isOn={followSelection}
+  offTooltipContent="Enable auto scroll"
```

Comment:
Wasn't immediately clear what auto scroll meant. Trying to think of alternatives that would succinctly capture "automatically scroll to directives/spans selected from the table that are offscreen"

Reply:
When I was thinking about it I was describing it as follow, scroll lock, sync, maybe there are other terms that are more intuitive

Reply:
I almost wonder if this should be an option in the table actions instead of the timeline?

Reply:
I'd like to keep it on the timeline if it's a toggle, since other views, like the problems view, may also set the selection. If it's not a toggle, and is instead a click action then perhaps we add it to the context menu.

Reply:
Good point.